documentation/architecture/12_DfE-Databricks.md

```diff
@@ -0,0 +1,137 @@
+# Decision - 0011 - Data pipeline Databricks conversion
+
+## Context and Problem Statement
+
+DfE's SQL server databases are being turned off and migrated to Databricks by the end of 2026. Our data pipelines consume some of this data, so we will have to adapt our pipelines to consume from Databricks. The presence of databricks allows an opportunity for updating of our pipelines - we could offload compute into databricks as an example of a good databricks pipeline for DfE.
+
+## Decision Drivers
+
+* SQL Server databases (including AnM) are being decommissioned by end of 2026
+* A lot of our source data including ancillary data is on Databricks already. More is scheduled to be added and we can request missing data.
+* The current SQL to extract data will need to be rewritten to Databricks SQL syntax as a Databricks job.
+* We will have access to Databricks compute. There are pre-allocated tokens allocated for funding, there are open questions around how this will be managed in the longer term.
+* The FBIT service can work from a csv pushed by a job from Databricks, or the Databricks API allows the FBIT application to trigger an export job.
+* Data export is currently tracked by Databricks unity catalogue, and is unrestricted.
+* Our data size is not large, the pipeline is not run often, and it doesn't take a lot of time to run. So the performance and scalability of the transformations in Databricks, while an improvement, isn't very relevant.
+
+## Considered Options
+
+A basic structure of the data pipeline illustrates the degrees of databricks involvement:
```

Comment:
nitpick: should be "Databricks" throughout

Now it's largely been ratified, I'll be pedantic about the casing.